snapshot/snapshot.go

```diff
@@ -1011,3 +1060,18 @@ func (o *snapshotter) snapshotRoot() string {
 func (o *snapshotter) snapshotDir(id string) string {
 	return filepath.Join(o.snapshotRoot(), id)
 }
+
+func treatAsProxyDriver(labels map[string]string) error {
```

Comment:
How about change the return type to `bool`?

Reply:
Done.